### ISSUE TITLE
IQ Integration is broken

### ISSUE THREAD

**Issue by madpah** [sonatype-nexus-community/bach#31]
Running `bach` in IQ mode produces an undefined method error:

```
php bach iq --file=composer.json --application=bach --host=***REDACTED*** --stage=build
 ____                         __
/\  _`\                      /\ \
\ \ \L\ \     __       ___   \ \ \___
 \ \  _ <'  /'__`\    /'___\  \ \  _ `\
  \ \ \L\ \/\ \L\.\_ /\ \__/   \ \ \ \ \
   \ \____/\ \__/.\_\\ \____\   \ \_\ \_\
    \/___/  \/__/\/_/ \/____/    \/_/\/_/


  _       _                          _   _
 /_)     /_` _  _  _ _/_     _  _   (/  /_` _ . _  _   _/  _
/_) /_/ ._/ /_// //_|/  /_/ /_//_' (_X /   / / /_'/ //_/ _\
    _/                  _/ /

In IQ.php line 65:

  Call to undefined method App\Parse\ComposerParser::get_packages()

```

**Comment by madpah** [comment]
See https://github.com/sonatype-nexus-community/bach/pull/33

**Comment by bhamail** [comment]
Fixed by #33
